app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,111 @@
+# rubocop:disable all
+

```

Comment:
RuboCop cops are meant to catch bad code and enforce best practices and the rubocop.yml file provided by [Microverse linters](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) repo doesn't enforce strict rules.

By disabling RuboCop rules, you are suppressing the potential warnings you would be getting, missing out on making your Ruby code more idiomatic. (See [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

Kindly remove line 1 and try to fix the rubocop linter error.